### PR TITLE
Automation tsekh

### PR DESCRIPTION
**Pull request by tsekhmeistruk**
Please, review this chunk of code

### CODE REVIEW

File: tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java [hunk 1]
```diff
@@ -50,4 +57,192 @@ public void checkAPIKeyPlaceholders(String apikey) {
...
+        String apiKey = countdownWidgetPage.getCountdownWidget().getAPIKeyTextField().getValue();
+        Serenity.getCurrentSession().put("apiKey", apiKey);
+    }
+
+    public void storeCurrentEventId() {
+        String eventId = countdownWidgetPage.getCountdownWidget().getEventIDTextField().getValue();
+        Serenity.getCurrentSession().put("eventId", eventId);
+    }
+
+    public void clickOnGetButton() {
```
Comment (quetzal77) [inline tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java]: Dont understand why so complex?

File: tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java [hunk 2]
```diff
@@ -50,4 +57,192 @@ public void checkAPIKeyPlaceholders(String apikey) {
...
+        WebElementFacade getCodeBtn = countdownWidgetPage.getCountdownWidget().getGetCodeButton();
+        getDriver().manage().window().maximize();
+        scrollToElement(getCodeBtn);
+        hoverToElement(getCodeBtn);
+        getCodeBtn.click();
+    }
+
+    public void thePopupEmbeddedCodeIsOpened() {
+        Assert.assertTrue(countdownWidgetPage.getCountdownWidget().getEmbeddedHtmlCode().isDisplayed());
```
Comment (quetzal77) [inline tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java]: ???????????

File: tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java [hunk 3]
```diff
@@ -50,4 +57,192 @@ public void checkAPIKeyPlaceholders(String apikey) {
...
+        try {
+            Thread.sleep(2000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void embeddedCodeContainsStoredApiKey() {
+        String embeddedApiKeyValue = countdownWidgetPage.getCountdownWidget().getEmbeddedApiKey();
```
Comment (quetzal77) [inline tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java]: ????????

File: tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java [hunk 4]
```diff
@@ -50,4 +57,192 @@ public void checkAPIKeyPlaceholders(String apikey) {
...
+        countdownWidgetPage.getCountdownWidget().getAPIKeyTextField().sendKeys(apiKey);
+    }
+
+    public void setEventId(String eventId) {
+        countdownWidgetPage.getCountdownWidget().getEventIDTextField().clear();
+        countdownWidgetPage.getCountdownWidget().getEventIDTextField().sendKeys(eventId);
+    }
+
+    public void clickResetButton() {
+        WebElementFacade resetBtn = countdownWidgetPage.getCountdownWidget().getResetButton();
```
Comment (quetzal77) [inline tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java]: ??????????

File: tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java [hunk 5]
```diff
@@ -50,4 +57,192 @@ public void checkAPIKeyPlaceholders(String apikey) {
...
+        countdownWidgetPage.getCountdownWidget().getSetThisId().click();
+    }
+
+    public void posterContainsKeyword(String keyword) {
+        String posterText = countdownWidgetPage.getCountdownWidget().getPosterWindow().getText();
+        Assert.assertTrue(String.format("The poster does not contains %s but has text: %s", keyword, posterText), keyword.toLowerCase().contains(posterText.toLowerCase()));
+    }
+
+    public void clickOnGetYourOwn() {
+        countdownWidgetPage.getCountdownWidget().getGetYourOwn().click();
```
Comment (quetzal77) [inline tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java]: ????

File: tests/serenity/src/test/resources/stories/site/products_and_docs/PD_Widget_Countdown.story [hunk 6]
```diff
@@ -10,76 +10,68 @@ Scenario: (/products-and-docs/widgets/countdown/) Verification for general page'
...
 
-Scenario: (/products-and-docs/widgets/countdown/) [TECHNICAL TAB] Check that required fields are not empty
+Scenario: (/products-and-docs/widgets/countdown/) TECHNICAL TAB - Check that required fields are not empty
 Given open Countdown Widget page
 Then the required fields are not empty on the Countdown Widget page
 
-Scenario: (/products-and-docs/widgets/countdown/) [TECHNICAL TAB] Verification of default embed code
+Scenario: (/products-and-docs/widgets/countdown/) TECHNICAL TAB - Verification of default embed code
 Given open Countdown Widget page
 And store ApiKey and EventId on Countdown Widget page
```
Comment (quetzal77) [inline tests/serenity/src/test/resources/stories/site/products_and_docs/PD_Widget_Countdown.story]: We have to discuss scenarious logic

File: tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java (hunk 3 above)
Comment (tsekhmeistruk) [inline tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java]: Please, could you provide more details what exactly '????' means?

File: tests/serenity/src/test/resources/stories/site/products_and_docs/PD_Widget_Countdown.story (hunk 6 above)
Comment (tsekhmeistruk) [inline tests/serenity/src/test/resources/stories/site/products_and_docs/PD_Widget_Countdown.story]: Ok

File: tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java (hunk 4 above)
Comment (tsekhmeistruk) [inline tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java]: Agree. removed redundant code. (forgot to delete after debugging)

File: tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java (hunk 2 above)
Comment (tsekhmeistruk) [inline tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java]: waiting while code will be uploaded

File: tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java (hunk 1 above)
Comment (tsekhmeistruk) [inline tests/serenity/src/main/java/bla/tm/steps/products_and_docs/PD_Widget_CountdownSteps.java]: forgot to delete after debugging. Removed redundant code

**Pull request merged**